services/QuillLMS/spec/workers/clean_active_activity_session_worker_spec.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CleanActiveActivitySessionWorker, type: :worker do
+  subject { described_class.new }
+
+  let(:question) { create(:question) }
+  let(:activity) { create(:activity, data: { 'questions' => [{ 'key' => question.uid }] }) }
+  let(:activity_session) { create(:activity_session, :started, activity:) }
+  let(:active_activity_session) { create(:active_activity_session, activity_session:) }
```

Comment:
It's not wrong to set up all this data, but given the actual tests in this file, could we just get away with creating `active_activity_session` and skip the other models we're establishhing?